VoxelSniperCore/src/main/java/com/github/kevindagame/command/VoxelBrushCommand.java

```diff
@@ -57,17 +57,21 @@ public boolean doCommand(IPlayer player, final String[] args) {
         // Command: /b <number> -- Change brush size
         try {
             int originalSize = snipeData.getBrushSize();
+            int newSize = Integer.parseInt(args[0]);
 
             var brush = sniper.getBrush(currentToolId);
-            if (brush == null) return false;
-
-            if (!new PlayerBrushSizeChangedEvent(player, currentToolId, brush, originalSize, snipeData.getBrushSize()).callEvent().isCancelled()) {
-                snipeData.setBrushSize(Integer.parseInt(args[0]));
-                snipeData.getVoxelMessage().size();
+            if (brush == null) {
+                snipeData.sendMessage(Messages.VOXEL_BRUSH_NO_PERMISSION);
```

Comment:
This doesn't seem right. Brush could be null but with the player having permission right (player can use all brushes but default brush for example)

Reply:
changed it to NO_BRUSH_SELECTED - https://github.com/KevinDaGame/VoxelSniper-Reimagined/commit/f0aef79c6e0f991864fae0861f297cf85584a1b6